lib/generators/rys/plugin/templates/lib/%name%/version.rb.tt

```diff
@@ -1,3 +1,5 @@
 module <%= camelized %>
+
```

Comment:
proč v routách si řádky odmazal a tady přidal?

Reply:
nelíbí se mi

Reply:
viz. nastavení rubocopu

Reply:
je to divné

Reply:
Jakto, že tady dáš nové řádky ale přímo v rys nové řádky nemáš

Reply:
rys jako takový jsem ti neupravoval. Jde mi o to co generujeme. Musel bych přepsal minimálně polovinu souborů :s